README.md

```diff
@@ -44,8 +44,9 @@ Building Xamarin.Android requires:
 * [Autotools (`autoconf`, `automake`, etc.)](#autotools)
 * [The Android SDK and NDK](#ndk)
 
-The `make prepare` build step will check that all required dependencies
-are present. If you would like `make prepare` to automatically install
+The `make prepare` build step (or `PrepareWindows.targets` on Windows) will
```

Comment:
It might be clearer to separate the **Build** section into Windows and Unix sections:

```markdown
# Build

Xamarin.Android can be built on Linux, macOS, and Windows.

## Linux and macOS

To build Xamarin.Android, ...

## Windows

To build Xamarin.Android, run:

    > msbuild build-tools\scripts\PrepareWindows.targets
    > msbuild Xamarin.Android.sln
```